src/prebid.js

```diff
@@ -272,6 +272,10 @@ function removeComplete() {
     .forEach(bid => responses.slice(responses.indexOf(bid), 1));
 }
 
+const videoAdUnit = adUnit => adUnit.mediaType === 'video';
+const nonVideoBidder = bid => !adaptermanager.videoAdapters.includes(bid.bidder);
+const hasNonVideoBidder = adUnit => adUnit.bids.filter(nonVideoBidder).length;
+
```

Comment:
These lines feel out of place here. Maybe create a video.js file and put them in there instead?